builders/json_serializable_fic/pubspec_overrides.yaml

```diff
@@ -0,0 +1,6 @@
+# melos_managed_dependency_overrides: json_serializable_fic_typehelpers,json_serializable_type_helper_utils
```

Comment:
please run `melos clean` before commiting or add **/pubspec_overrides.yaml to .gitignore so that the overrides are not commited